### PR TITLE
add content for updates

### PR DESCRIPTION
**Pull request by sevigrimm**

### CODE REVIEW

File: best-practices/content/50_Updates.tex [hunk 1]
```diff
@@ -0,0 +1,130 @@
+\chapter{Updates}
+\section{Einleitung}
+Updates gehören zu den effizientesten Möglichkeiten, sich gegen Angreifer zu schützen.
```
Comment (marcom4rtinez) [inline best-practices/content/50_Updates.tex:3]:
```suggestion
Updates gehören zu den effektivsten Möglichkeiten, sich gegen Angreifer zu schützen.
```

File: best-practices/content/50_Updates.tex [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+Für Angreifer ist es ein Goldenes Los, wenn sie Software entdecken, welche nicht aktualisiert wurde und tendenziell sogar eine \acrfull{cve}\footnote{Link: https://www.cve.org/} dazu existiert.\\
```
Comment (marcom4rtinez) [inline best-practices/content/50_Updates.tex:4]:
```suggestion
Für Angreifer ist es ein goldenes Los Software zu entdecken, welche nicht aktualisiert wurde und tendenziell sogar ein \acrfull{cve}\footnote{Link: https://www.cve.org/} dazu existiert.\\
```

File: best-practices/content/50_Updates.tex [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+
+Daher ist es wichtig, möglichst Zeitnah neue Updates zu installieren.
```
Comment (marcom4rtinez) [inline best-practices/content/50_Updates.tex:6]:
```suggestion
Daher ist es wichtig, möglichst zeitnah aktuelle Updates zu installieren.
```

File: best-practices/content/50_Updates.tex [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+Viele Softwarehersteller bieten eine automatische Updatefunktion, welche die neusten Updates direkt installiert.
+
+\section{Windows Updates}
+Windows Updates können direkt in den Systemeinstellungen heruntergeladen und installiert werden.
+Dies muss jedoch standardmässig von den Benutzern gemacht werden.
+Diese Updates können über die Group Policy durchgesetzt werden, damit sichergestellt werden kann, dass die System immer auf neustem Stand sind.\\
```
Comment (marcom4rtinez) [inline best-practices/content/50_Updates.tex:12]:
```suggestion
Diese Updates können über die Group Policy erzwungen werden, damit sichergestellt werden kann, dass die System immer auf aktuellen Stand sind.\\
```

File: best-practices/content/50_Updates.tex [hunk 5]
```diff
@@ -0,0 +1,130 @@
...
+
+Wichtige Anmerkung:
+\begin{lstlisting}
+Bei kritischen Systemen muss immer zuerst überprüft werden, ob alle Funktionen nach dem Update noch funktionieren.
+\end{lstlisting}
+
+\subsection{GPO für Clients}
+Falls kritische Clients vorhanden sind, sollte \acrfull{wsus} verwendet werden. Siehe Kapitel \hyperref[subsec:wsus]{WSUS}\\
+
+Im Group Policy Management muss eine neue Group Policy erstellt werden, welche mit der OU verknüpft ist, die alle Windows Geräte enthält.
```
Comment (marcom4rtinez) [inline best-practices/content/50_Updates.tex:22]: maybe glossary for OU

File: best-practices/content/50_Updates.tex [hunk 6]
```diff
@@ -0,0 +1,130 @@
...
+Daher ist es wichtig, möglichst Zeitnah neue Updates zu installieren.
+Viele Softwarehersteller bieten eine automatische Updatefunktion, welche die neusten Updates direkt installiert.
+
+\section{Windows Updates}
+Windows Updates können direkt in den Systemeinstellungen heruntergeladen und installiert werden.
+Dies muss jedoch standardmässig von den Benutzern gemacht werden.
+Diese Updates können über die Group Policy durchgesetzt werden, damit sichergestellt werden kann, dass die System immer auf neustem Stand sind.\\
+
+Wichtige Anmerkung:
+\begin{lstlisting}
```
Comment (marcom4rtinez) [inline best-practices/content/50_Updates.tex:15]: style looks awefull
image


Maybe do
```
\textbf{Wichtige Anmerkung:} 
Bei kritischen Systemen muss immer zuerst überprüft werden, ob alle Funktionen nach dem Update noch funktionieren.
```

File: best-practices/content/50_Updates.tex [hunk 7]
```diff
@@ -0,0 +1,130 @@
...
+\end{minipage}
+\begin{minipage}{0.5\linewidth}
+    \begin{figure}[H]
+        \centering
+        \includegraphics[width=\linewidth]{../img/Updates/client-force-restart-deadline.png}
+        \caption{GPO Client Updates 4}
+    \end{figure}
+\end{minipage}\\
+
+Mit diesen Einstellungen werden die Updates am Morgen heruntergeladen und installiert.
```
Comment (marcom4rtinez) [inline best-practices/content/50_Updates.tex:62]:
```suggestion
Mit diesen Einstellungen werden die Updates am Morgen, um 05:00 Uhr,  heruntergeladen und installiert.
```

File: best-practices/content/50_Updates.tex [hunk 8]
```diff
@@ -0,0 +1,130 @@
...
+Die Benutzer müssen spätestens nach 7 Tagen ihre Computer neu starten.
+Ausserdem werden die Computer nur ausserhalb der definierten Arbeitszeit automatisch neu gestartet.\\
+
+Diese Einstellungen sollen nicht strikt übernommen werden, sondern den Bedürfnissen des Betriebs angepasst werden.
+Wichtig ist jedoch, dass die Updates automatisch installiert werden und das die Computer nach Updates neugestartet werden müssen.
+
+\subsection{GPO für Server}
+Falls kritische Clients vorhanden sind, sollte \acrfull{wsus} verwendet werden. Siehe Kapitel \hyperref[subsec:wsus]{WSUS}\\
```
Comment (marcom4rtinez) [inline best-practices/content/50_Updates.tex:70]:
```suggestion
Falls kritische Server vorhanden sind, sollte \acrfull{wsus} verwendet werden. Siehe Kapitel \hyperref[subsec:wsus]{WSUS}\\
```

File: best-practices/content/50_Updates.tex [hunk 9]
```diff
@@ -0,0 +1,130 @@
...
+
+Updates für Server sollten installiert werden, wenn Sie nicht in Verwendung sind, da sie eventuell für kurze Zeit nicht verfügbar sind.
```
Comment (marcom4rtinez) [inline best-practices/content/50_Updates.tex:72]:
```suggestion
Updates für Server sollten installiert werden, wenn Sie nicht in Verwendung sind, da sie für kurze Zeit nicht verfügbar sein werden.
```

File: best-practices/content/50_Updates.tex [hunk 10]
```diff
@@ -0,0 +1,130 @@
...
+Daher ist die Group Policy für Server anderst als für Clients.
+Für Server die 24/7h verwendet werden, muss ein ``Maintenance Window'' gesetzt und alle Benutzer informiert werden.\\
```
Comment (marcom4rtinez) [inline best-practices/content/50_Updates.tex:74]:
```suggestion
Für Server die 24/7 verwendet werden, muss ein ``Maintenance Window'' gesetzt und alle Benutzer informiert werden.\\
```

File: best-practices/content/50_Updates.tex [hunk 11]
```diff
@@ -0,0 +1,130 @@
...
+
+Im Group Policy Management muss eine neue Group Policy erstellt werden, welche mit der OU verknüpft ist, die alle Windows Geräte enthält.
```
Comment (marcom4rtinez) [inline best-practices/content/50_Updates.tex:76]: OU glossary maybe

File: best-practices/content/50_Updates.tex [hunk 12]
```diff
@@ -0,0 +1,130 @@
...
+        \centering
+        \includegraphics[width=\linewidth]{../img/Updates/client-allow-immediate-updates.png}
+        \caption{GPO Client Updates 1}
+    \end{figure}
+\end{minipage}
+\begin{minipage}{0.5\linewidth}
+    \begin{figure}[H]
+        \centering
+        \includegraphics[width=\linewidth]{../img/Updates/server-configure-automatic-updates.png}
+        \caption{GPO Client Updates 2}
```
Comment (marcom4rtinez) [inline best-practices/content/50_Updates.tex:98]:
```suggestion
        \caption{GPO Server Updates 2}
```

File: best-practices/content/50_Updates.tex [hunk 13]
```diff
@@ -0,0 +1,130 @@
...
+
+Mit \textbf{Rechtsklick $\rightarrow$ Edit} kann die neue Group Policy bearbeitet werden.
+Unter \textbf{Computer Configuration $\rightarrow$ Policies $\rightarrow$ Administrative Templates $\rightarrow$ Windows Components $\rightarrow$ Windows Update} findet man alle Policies bezüglich Windows Updates.
+Für Server werden folgende Policies eingerichtet:
+
+\begin{minipage}{0.5\linewidth}
+    \begin{figure}[H]
+        \centering
+        \includegraphics[width=\linewidth]{../img/Updates/client-allow-immediate-updates.png}
+        \caption{GPO Client Updates 1}
```
Comment (marcom4rtinez) [inline best-practices/content/50_Updates.tex:91]:
```suggestion
        \caption{GPO Server Updates 1}
```

File: best-practices/content/50_Updates.tex [hunk 14]
```diff
@@ -0,0 +1,130 @@
...
+        \centering
+        \includegraphics[width=\linewidth]{../img/Updates/server-configure-automatic-updates.png}
+        \caption{GPO Client Updates 2}
+    \end{figure}
+\end{minipage}\\
+\begin{minipage}{0.5\linewidth}
+    \begin{figure}[H]
+        \centering
+        \includegraphics[width=\linewidth]{../img/Updates/client-auto-restart-during-working-hours.png}
+        \caption{GPO Client Updates 3}
```
Comment (marcom4rtinez) [inline best-practices/content/50_Updates.tex:105]:
```suggestion
        \caption{GPO Server Updates 3}
```

File: best-practices/content/50_Updates.tex [hunk 15]
```diff
@@ -0,0 +1,130 @@
...
+    \end{figure}
+\end{minipage}
+\begin{minipage}{0.5\linewidth}
+    \begin{figure}[H]
+        \centering
+        \includegraphics[width=\linewidth]{../img/Updates/server-automatic-restart.png}
+        \caption{GPO Client Updates 4}
```
Comment (marcom4rtinez) [inline best-practices/content/50_Updates.tex:112]:
```suggestion
        \caption{GPO Server Updates 4}
```

File: best-practices/content/50_Updates.tex [hunk 16]
```diff
@@ -0,0 +1,130 @@
...
+    \end{figure}
+\end{minipage}\\
+
+Die Updates werden auf den Servern in der Nacht installiert und die Server anschliessend neu gestartet.
```
Comment (marcom4rtinez) [inline best-practices/content/50_Updates.tex:116]: specify Nacht

Comment (sevigrimm) [inline best-practices/content/50_Updates.tex:116]:
```suggestion
Die Updates werden auf den Servern in der Nacht, um 01:00 Uhr, installiert und die Server anschliessend neu gestartet.
```

**Pull request merged**